make/upgrade

```diff
@@ -8,16 +8,24 @@ set -o errexit -o nounset
 
 NAMESPACE="cf"
 UAA_NAMESPACE="uaa"
-UAA_CA_CERT=internal-ca-cert
 
 GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "${GIT_ROOT}"
 
-# Determine the kube revision of the chart controlling the specified namespace.
-release_version() { helm list --date --reverse --max 1 --namespace "$1" | awk '{ print $2 }' | tail -n 1 ; }
+get_uaa_secret_name() {
+    kubectl get pods \
+	--namespace "${UAA_NAMESPACE}" \
+	-o jsonpath='{.items[*].spec.containers[?(.name=="uaa")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}'
+}
+
+get_uaa_secret_key() {
+    kubectl get pods \
+	--namespace "${UAA_NAMESPACE}" \
+	-o jsonpath='{.items[*].spec.containers[?(.name=="uaa")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.key}'
+}
 
 get_uaa_secret () {
```

Comment:
This function also should take the env variable name as an argument, and not the name of the secret key. Instead it should call `$(get_uaa_secret_key $ENV_NAME)` itself.

Reply:
Done with commit 405a6bdb262d41482dfb7d8e5045cf3f406aa183